ghp_import.py

```diff
@@ -216,6 +222,39 @@ def options():
     ]
 
 
+def ghp_import(srcdir, **kwargs):
+    if not os.path.isdir(srcdir):
+        raise GhpError("Not a directory: %s" % srcdir)
+
+    opts = {
+        'remote': 'origin',
+        'branch': 'gh-pages',
+        'mesg': 'Update documentation',
+        'push': False,
+        'force': False,
+        'use_shell': False,
+        'followlinks': False,
+        'cname': None,
+        'nojekyll': False
+    }
+
+    opts.update(kwargs)
+
+    git = Git(use_shell=opts['use_shell'])
+    git.check_repo()
+
+    if not git.try_rebase(opts['remote'], opts['branch']):
+        raise GhpError("Failed to rebase %s branch." % opts['branch'])
+
+    run_import(git, srcdir, **opts)
+
+    if push:
+        if force:
```

Comment:
Maybe I'm missing something, but where are these defined? Should these reference `opts`?

```suggestion
    if opts['push']:
        if opts['force']:
```